src/index.html

```diff
@@ -13,5 +13,145 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="http://localhost:3000/api" method="post"
+    onsubmit="onSubmit()">
+      <fieldset class="block">
+        <legend> Personal information</legend>
+        <label class="form-field" for="surname">
+          Surname:
+          <input type="text"
+          name="surname" id="surname"
+          required>
```

Comment:
```suggestion
          <input 
            type="text"
            name="surname"
            id="surname"
            required
          >
```
Форматируй теги таким образом